client/rest/src/plugins/metadata/metal.js

```diff
@@ -33,8 +33,8 @@ export const metal = {
 		if (!input.length)
 			throw new Error('input must not be empty');
 		const hashedBytes = sha3_256(input);
-		const uint64 = catapult.utils.uint64.fromBytes(hashedBytes.subarray(0, 8));
-		return [uint64[0], uint64[1] & 0x7FFFFFFF];
+		const uint64 = utils.bytesToBigInt(hashedBytes.subarray(0, 8), 8);
+		return uint64 & 0x7FFFFFFFFFFFFFFFn;
```

Comment:
can remove from the return message ``, represented as an array of two numbers.``